### ISSUE TITLE
"Weights only load failed" error, when running the workflow.

### ISSUE THREAD

**Issue by RayEbb** [vantagewithai/Vantage-Nodes#2]
After generated  a few speakers, i wanted to try the Speaker Generation group. But when i wanted to run the workflow, this error appeared.

<img width="1668" height="561" alt="Image" src="https://github.com/user-attachments/assets/3086bf6d-b359-4cb6-87f1-8e969c4b589b" />

**Comment by vantagewithai** [comment]
Could you please paste the output of 

`pip list`

I’m running this with PyTorch 2.10 and not seeing any errors, so that information would help narrow down what’s different on your setup.

**Comment by RayEbb** [comment]
I'm using the latest ComfyUI Portable. I forgot to mention this in my first post.

This is the Output:

Package                                Version
-------------------------------------- ------------------------------------
absl-py                                2.3.1
accelerate                             1.12.0
addict                                 2.4.0
aiofiles                               24.1.0
aiohappyeyeballs                       2.6.1
aiohttp                                3.13.3
aiohttp_socks                          0.11.0
aiosignal                              1.4.0
aiosqlite                              0.22.1
albucore                               0.0.24
albumentations                         2.0.8
alembic                                1.18.1
annotated-types                        0.7.0
antlr4-python3-runtime                 4.9.3
anyio                                  4.12.1
attrs                                  25.4.0
audioread                              3.1.0
av                                     16.0.1
beautifulsoup4                         4.14.3
bitsandbytes                           0.49.1
certifi                                2026.1.4
cffi                                   2.0.0
chardet                                5.2.0
charset-normalizer                     3.4.4
click                                  8.3.1
clip                                   1.0
clip-interrogator                      0.6.0
color-matcher                          0.6.0
colorama                               0.4.6
coloredlogs                            15.0.1
colorlog                               6.10.1
comfy-kitchen                          0.2.7
comfyui-embedded-docs                  0.4.0
comfyui_frontend_package               1.37.11
comfyui_workflow_templates             0.8.27
comfyui-workflow-templates-core        0.3.115
comfyui-workflow-templates-media-api   0.3.47
comfyui-workflow-templates-media-image 0.3.75
comfyui-workflow-templates-media-other 0.3.98
comfyui-workflow-templates-media-video 0.3.42
contourpy                              1.3.3
cryptography                           46.0.3
cycler                                 0.12.1
ddt                                    1.7.2
decorator                              5.2.1
decord                                 0.6.0
deepdiff                               8.6.1
diffusers                              0.36.0
diskcache                              5.6.3
docutils                               0.22.4
easydict                               1.13
einops                                 0.8.1
facexlib                               0.3.0
filelock                               3.20.0
filterpy                               1.4.5
flatbuffers                            25.12.19
flet                                   0.80.4
fonttools                              4.61.1
frozenlist                             1.8.0
fsspec                                 2025.12.0
ftfy                                   6.3.1
fvcore                                 0.1.5.post20221221
gdown                                  5.2.1
gguf                                   0.17.1
gitdb                                  4.0.12
GitPython                              3.1.46
greenlet                               3.3.1
groundingdino-py                       0.4.0
h11                                    0.16.0
h2                                     4.3.0
hf-xet                                 1.2.0
hpack                                  4.1.0
httpcore                               1.0.9
httpx                                  0.28.1
huggingface-hub                        0.36.0
humanfriendly                          10.0
hydra-core                             1.3.2
hyperframe                             6.1.0
idna                                   3.11
ImageIO                                2.37.2
imageio-ffmpeg                         0.6.0
importlib_metadata                     8.7.1
iopath                                 0.1.10
Jinja2                                 3.1.6
joblib                                 1.5.3
jsonschema                             4.26.0
jsonschema-specifications              2025.9.1
kiwisolver                             1.4.9
kornia                                 0.8.2
kornia_rs                              0.1.10
lark                                   1.3.1
lazy_loader                            0.4
librosa                                0.11.0
llama_cpp_python                       0.3.18
llvmlite                               0.46.0
lxml                                   6.0.2
Mako                                   1.3.10
manifold3d                             3.3.2
mapbox_earcut                          2.0.0
markdown-it-py                         4.0.0
MarkupSafe                             2.1.5
matplotlib                             3.10.8
matrix-nio                             0.25.2
mdurl                                  0.1.2
mediapipe                              0.10.32
ml_dtypes                              0.5.4
mpmath                                 1.3.0
msgpack                                1.1.2
mss                                    10.1.0
multidict                              6.7.0
natsort                                8.4.0
networkx                               3.6.1
numba                                  0.63.1
numpy                                  2.3.5
nunchaku                               1.2.0+torch2.9
nvidia-ml-py                           13.590.48
oauthlib                               3.3.1
olefile                                0.47
omegaconf                              2.3.0
onnx                                   1.20.1
onnxruntime                            1.23.2
onnxruntime-gpu                        1.23.2
open_clip_torch                        3.2.0
opencv-contrib-python                  4.13.0.90
opencv-python                          4.13.0.90
opencv-python-headless                 4.13.0.90
orderly-set                            5.5.0
packaging                              26.0
pathspec                               1.0.3
peft                                   0.18.1
piexif                                 1.1.3
pillow                                 12.1.0
pip                                    25.3
platformdirs                           4.5.1
pooch                                  1.8.2
portalocker                            3.2.0
propcache                              0.4.1
protobuf                               5.29.5
psutil                                 7.2.1
py-cpuinfo                             9.0.0
pycocotools                            2.0.11
pycollada                              0.9.3
pycparser                              3.0
pycryptodome                           3.23.0
pydantic                               2.12.5
pydantic_core                          2.41.5
pydantic-settings                      2.12.0
pygit2                                 1.19.1
PyGithub                               2.8.1
Pygments                               2.19.2
PyJWT                                  2.10.1
pyloudnorm                             0.2.0
PyMatting                              1.1.14
PyNaCl                                 1.6.2
pynvml                                 13.0.1
pyparsing                              3.3.2
pyreadline3                            3.5.4
PySocks                                1.7.1
python-dateutil                        2.9.0.post0
python-dotenv                          1.2.1
python-socks                           2.8.0
pywin32                                311
PyYAML                                 6.0.3
referencing                            0.37.0
regex                                  2026.1.15
rembg                                  2.0.72
repath                                 0.9.0
requests                               2.32.5
rich                                   14.3.1
rpds-py                                0.30.0
rtree                                  1.4.1
safetensors                            0.7.0
sageattention                          2.2.0+cu130torch2.9.0andhigher.post4
scikit_build_core                      0.11.6
scikit-image                           0.26.0
scikit-learn                           1.8.0
scipy                                  1.17.0
segment-anything                       1.0
sentencepiece                          0.2.1
setuptools                             70.2.0
shapely                                2.1.2
shellingham                            1.5.4
simsimd                                6.5.12
six                                    1.17.0
smmap                                  5.0.2
sounddevice                            0.5.5
soundfile                              0.13.1
soupsieve                              2.8.3
sox                                    1.5.0
soxr                                   1.0.0
spandrel                               0.4.1
SQLAlchemy                             2.0.46
stringzilla                            3.12.6
supervision                            0.6.0
svg.path                               7.0
sympy                                  1.14.0
tabulate                               0.9.0
termcolor                              3.3.0
threadpoolctl                          3.6.0
tifffile                               2026.1.14
tiktoken                               0.12.0
timm                                   1.0.24
tokenizers                             0.22.2
toml                                   0.10.2
tomli                                  2.4.0
torch                                  2.9.1+cu130
torchaudio                             2.9.1+cu130
torchsde                               0.2.6
torchvision                            0.24.1+cu130
tqdm                                   4.67.1
trampoline                             0.1.2
transformers                           4.57.6
transparent-background                 1.3.4
trimesh                                4.11.1
triton-windows                         3.5.1.post24
typer                                  0.21.1
typing_extensions                      4.15.0
typing-inspection                      0.4.2
unpaddedbase64                         2.1.0
urllib3                                2.6.3
uv                                     0.9.7
vhacdx                                 0.0.10
watchdog                               6.0.0
wcwidth                                0.3.5
wget                                   3.2
xxhash                                 3.6.0
yacs                                   0.1.8
yapf                                   0.43.0
yarl                                   1.22.0
zipp                                   3.23.0

**Comment by Strix-3D** [comment]
I used this to test the new Kimi ai model. Fixed by replacing line 6 of Comfyui\ComfyUI\custom_nodes\Vantage-Nodes\qwen_tts_speaker_bank.py

`from .qwen_tts_voice_storage import get_custom_voice_root`

 with :

```
from .qwen_tts_voice_storage import get_custom_voice_root
from .qwen_tts.inference.qwen3_tts_model import VoiceClonePromptItem
torch.serialization.add_safe_globals([VoiceClonePromptItem])
```

**Comment by RayEbb** [comment]
> I used this to test the new Kimi ai model. Fixed by replacing line 6 of Comfyui\ComfyUI\custom_nodes\Vantage-Nodes\qwen_tts_speaker_bank.py
> 
> `from .qwen_tts_voice_storage import get_custom_voice_root`
> 
> with :
> 
> ```
> from .qwen_tts_voice_storage import get_custom_voice_root
> from .qwen_tts.inference.qwen3_tts_model import VoiceClonePromptItem
> torch.serialization.add_safe_globals([VoiceClonePromptItem])
> ```

Thank you very much! It's running now without any errors!

**Comment by vantagewithai** [comment]
It looks like this issue is only happening on PyTorch 2.6 to 2.9.1. On PyTorch 2.10, it works fine and doesn’t even require adding a safe global. I’ll update this in the repo.

Going forward, I guess I should test the node across multiple PyTorch versions before releasing it.
